.github/workflows/unsafe-reviewers.yml

```diff
@@ -8,7 +8,7 @@ on:
 permissions:
   id-token: write
   contents: read
-  pull-requests: read
+  pull-requests: write 
```

Comment:
nit: is there a trailing space here or is it just GH being weird?

Reply:
It was indeed a trailing space. Just pushed a change to remove it.

Reply:
Thanks. I recommend configuring your editor to remove trailing spaces on save.